src/linker/Linker.Dataflow/CompilerGeneratedState.cs

```diff
@@ -145,7 +145,7 @@ void ProcessMethod (MethodDefinition method)
 				// Discover calls or references to lambdas or local functions. This includes
 				// calls to local functions, and lambda assignments (which use ldftn).
 				if (method.Body != null) {
-					foreach (var instruction in _context.MethodBodyInstructionsProvider.GetMethodBody (method.Body).Instructions) {
+					foreach (var instruction in _context.GetMethodIL (method.Body).Instructions) {
```

Comment:
Wouldn't be better to replace all `GetMethodIL (method.Body)` calls with `GetMethodIL (method)`